heron/statemgrs/src/java/com/twitter/heron/statemgr/zookeeper/curator/CuratorStateManager.java

```diff
@@ -79,8 +79,8 @@ public void initialize(Config newConfig) {
 
       String newConnectionString = tunneledResults.first;
       if (newConnectionString.isEmpty()) {
-        throw new IllegalArgumentException("Cannot connect to tunnelHost: "
-            + tunnelConfig.getTunnelHost() + " Bad connectionString: " + connectionString);
+        throw new IllegalArgumentException("Cannot connect to tunnel host: "
+            + tunnelConfig.getTunnelHost());
```

Comment:
No. And I checked this part of the code. Whether we can connect to the tunnel is independent from `connectionString`.  We will [use `connectionString`](https://github.com/twitter/heron/pull/1571/files/c43c5c613fe450874a130944a3509233e67f1ca3#diff-485c69d31fdc231bde091e0adfaad38dR92) later